pkg/utils/helper.go

```diff
@@ -44,10 +44,9 @@ func FormatSize(size int64) string {
 	return fmt.Sprintf("%.2fMiB", mbSize)
 }
 
+// ValidateUserName checks if the username is valid by length and allowed characters.
 func ValidateUserName(username string) bool {
-	pattern := `^[a-zA-Z0-9]{1,255}$`
-	re := regexp.MustCompile(pattern)
-	return re.MatchString(username)
+	return len(username) >= 1 && len(username) <= 255 && !strings.ContainsAny(username, `,"~#%$`)
```

Comment:
Yes, it meets the username validators, but it also satisfy certain requirements outlined in the issue #195.

Reply:
Just to confirm, should I add space trimming and an empty-check to the username validation here to fully meet issue #195 requirements, or handle that separately?

Reply:
I think it would be better to validate and trim spaces in the input fields here.

Reply:
@JianMinTang @bupd  is it fine now?